ow_lander/scripts/trajectory_publisher.py

```diff
@@ -98,30 +100,30 @@ def talker(req):
 
   # Use two files instead of one if this is a guarded move
   if filename.startswith('guarded_move_traj_'):
-    # reading csv file 
+    # reading csv file
     guard_filename = filename
     prefix = "pre_"
     filename = prefix + filename
     guarded_move_bool = True
 
-  # Reading csv file 
-  with open(filename, 'r') as csvfile: 
-    # creating a csv reader object 
-    csvreader = csv.reader(csvfile) 
+  # Reading csv file
```

Comment:
Is your editor modifying whitespace on its own? There are a lot of changes here that aren't really changes. It's better to avoid that if possible.

Reply:
Mhm, I am not sure how that happened...